ebl/v3/EBL_v3.0.0-Beta-2.yaml

```diff
@@ -2526,14 +2638,18 @@ components:
             Indicates the shipper’s platform on which the eBL should be issued. The value **MUST** be one of:
             - `WAVE` (Wave)
             - `CARX` (CargoX)
-            - `ICED` (ICE Digital)
+            - `ESSD` (EssDOCS)
+            - `IDT` (ICE Digital Trade)
             - `BOLE` (Bolero)
             - `EDOX` (EdoxOnline)
             - `IQAX` (IQAX)
             - `SECR` (Secro)
-            - `TRGO` (TradeGo)
+            - `TRGO` (TradeGO)
+            - `ETEU` (eTEU)
 
             **Condition:** Mandatory for electronic Bill of Lading (`isElectronic=TRUE`).
```

Comment:
"In the old days", you used `=TRUE` in all caps.

But in the recently changed parts you use `=true` (or `=false)` in lower case. Should we be consistent here?

Reply:
I agree - fixed